res/skins/LateNight/style.qss

```diff
@@ -1205,7 +1205,8 @@ WEffectSelector {
     border: 1px solid #666;
     border-radius: 2px;
     padding: 0px;
-    margin: 0px 0px 0px -7px;
+    /* hack around text getting cut off with scaled checkbox on macOS with Retina screens */
+    margin: 0px 0px 0px -24px;
```

Comment:
@beenisss could you try making -24px here a little more negative until no effect names get cut off?

Reply:
Here's how it looks at -26, -28 and -30:

-26
<img width="187" alt="margin -26" src="https://user-images.githubusercontent.com/36423450/38245428-f42116f0-3735-11e8-9eab-d7b9a84bdfb7.png">

-28
<img width="178" alt="margin -28" src="https://user-images.githubusercontent.com/36423450/38245442-01684900-3736-11e8-9b49-89a73abedd97.png">

-30
<img width="168" alt="margin -30" src="https://user-images.githubusercontent.com/36423450/38245448-05a6f48a-3736-11e8-9015-d9244b178dfe.png">